src/test/groovy/org/prebid/server/functional/tests/EidsSpec.groovy

```diff
@@ -91,6 +94,58 @@ class EidsSpec extends BaseSpec {
         assert !bidderRequest.user.eids
     }
 
+    def "PBS eids shouldn't include warning for unknown bidder in request"() {
```

Comment:
`PBS eids shouldn't include warning for unknown bidder when test and debug disabled`